src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -25,33 +26,36 @@ public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
                 new Address("Blk 30 Geylang Street 29, #06-40"),
-                getTagSet("friends")),
+                getTagSet("Celebrity")),
```

Comment:
I like how you have changed the context of our sample data to fit our application!